MainProject/Factory.cs

```diff
@@ -0,0 +1,101 @@
+﻿using ClassLibrary;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Factory iTechArt = new Factory
+        (
+            "iTechArt",
+            "Krakow",
+            1,
+            new Employee[]
+            {
+                new Employee ("John", "Doe", 23),
+                new Employee ("Helena", "Bug", 20),
+                new Employee ("Harry", "Potter", 19),
+                new Employee ("Hanna", "Smith", 27, 5, "Middle"),
+                new Employee ("Clark", "Kent", 30, 10, "Senior" ),
+                new Employee ("Elon", "Musk", 22, 1, "Junior"),
+                new Employee ( "Barak", "Obama", 35, 15, "Senior"),
```

Comment:
Additional task:
1. add new class Manager inherited from Employee. Add to Manager class a list of subordinate workers and a method for adding them
2. Create object of Manager class. Add some emplyee to him
3. Add the manager to iTechArt Factory with whole information

Reply:
done, but let's check, feel not confident if I done what's expected:)